internal/pipelines/check-embargoed-cves/check-embargoed-cves.yaml

```diff
@@ -4,7 +4,7 @@ kind: Pipeline
 metadata:
   name: check-embargoed-cves
   labels:
-    app.kubernetes.io/version: "0.1.0"
+    app.kubernetes.io/version: "0.2.0"
```

Comment:
It's a breaking change - `taskGitRevision` is required now. So should be `1.0.0`.

Reply:
Updated